build/CMakeCache.txt

```diff
@@ -0,0 +1,378 @@
+# This is the CMakeCache file.
```

Comment:
@SundarKrishnakumar you shouldn't check in the build directory.  This should be ignored in your .gitignore.  See https://github.com/cu-ecen-aeld/aesd-assignments/blob/master/.gitignore#L4..  I'm curious how/why you added it?

Please remove it.

Also pay close attention to commits which add thousands of files like this one: https://github.com/cu-ecen-aeld/assignment-1-SundarKrishnakumar/commit/2d497ec88dce87bc65f993c6960474170a6e1125#diff-44575cf5b28512d75644bf54a517dcef304ff809fd511747621b4d64f19aac66  - it probably means you are adding stuff you don't want to add.